src/ui/common/src/components/layouts/NavBar.tsx

```diff
@@ -0,0 +1,219 @@
+  NotificationStatus,
+} from '../../utils/notifications';
+import NotificationsPopover from '../notifications/NotificationsPopover';
+import styles from './menu-sidebar-styles.module.css';
+import { MenuSidebarWidthNumber } from './menuSidebar';
+
+const pathPrefix = getPathPrefix();
+
+export class BreadcrumbLinks {
+  static readonly HOME = new BreadcrumbLinks(`${pathPrefix}/`, 'Home');
+  static readonly DATA = new BreadcrumbLinks(`${pathPrefix}/data`, 'Data');
+  static readonly INTEGRATIONS = new BreadcrumbLinks(
+    `${pathPrefix}/integrations`,
+    'Integrations'
+  );
+  static readonly WORKFLOWS = new BreadcrumbLinks(
+    `${pathPrefix}/workflows`,
+    'Workflows'
+  );
+  static readonly ACCOUNT = new BreadcrumbLinks(
+    `${pathPrefix}/account`,
+    'Account'
+  );
+  static readonly ERROR = new BreadcrumbLinks(
+    `${pathPrefix}/404`,
+    'Page Not Found'
+  );
+
+  constructor(public readonly address: string, public readonly name: any) {}
+
+  toString() {
+    return this.name;
+  }
+}
+
+/**
+ * The `NavBar` is the core sidebar that we include throughout our UI. It
+ * is pinned to the top of every page in our UI, and it includes
+ * information about the site hierarchy, notifications, and settings/accounts page.
+ */
+const NavBar: React.FC<{
+  user: UserProfile;
+  breadcrumbs: BreadcrumbLinks[];
+}> = ({ user, breadcrumbs }) => {
+  const [userPopoverAnchorEl, setUserPopoverAnchorEl] = useState(null);
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  console.log(breadcrumbs);
+
+  const userPopoverOpen = Boolean(userPopoverAnchorEl);
+  const open = Boolean(anchorEl);
+
+  const numUnreadNotifications = useSelector(
+    (state: RootState) =>
+      state.notificationsReducer.notifications.filter(
+        (notification) =>
+          notification.level !== NotificationLogLevel.Success &&
+          notification.status === NotificationStatus.Unread
+      ).length
+  );
+
+  const handleUserPopoverClick = (event: React.MouseEvent) => {
+    setUserPopoverAnchorEl(event.currentTarget);
+  };
+
+  const handleCloseUserPopover = () => {
+    setUserPopoverAnchorEl(null);
+  };
+
+  const handleClick = (event: React.MouseEvent<HTMLDivElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const notificationsPopoverId = open ? 'simple-popover' : undefined;
+  const userPopoverId = userPopoverOpen ? 'user-popover' : undefined;
+
+  const avatarStyling = { width: '24px', height: '24px', marginLeft: '16px' };
+
+  const avatar = user.picture ? (
+    <Avatar
+      className={styles['user-avatar']}
+      sx={avatarStyling}
+      src={user.picture}
+      onClick={handleUserPopoverClick}
+    />
+  ) : (
+    <Avatar
+      className={styles['user-avatar']}
+      sx={avatarStyling}
+      onClick={handleUserPopoverClick}
+    >
+      {user.name !== 'aqueduct user' ? user.name : null}
+    </Avatar>
+  );
+
+  /* Header */
```

Comment:
Which part is the header? the whole component below?

Or is it both the header and the sidebar?